apps/mobile/src/features/ConfirmTx/components/ConfirmationView/ConfirmationView.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import {
+  CustomTransactionInfo,
+  MultisigExecutionDetails,
+  SettingsChangeTransaction,
+  TransactionData,
+  TransactionDetails,
+  TransferTransactionInfo,
+} from '@safe-global/store/gateway/AUTO_GENERATED/transactions'
+import { QueueSummary } from '../confirmation-views/QueueSummary'
+import { AddSigner } from '../confirmation-views/AddSigner'
+import { ETxType } from '@/src/types/txType'
+import { getTransactionType } from '@/src/utils/transactions'
+import { Contract } from '../confirmation-views/Contract'
+import { SendNFT } from '../confirmation-views/SendNFT'
+import { SwapOrder } from '../confirmation-views/SwapOrder'
+import { OrderTransactionInfo } from '@safe-global/store/gateway/types'
+import { RemoveSigner } from '../confirmation-views/RemoveSigner'
+import { GenericView } from '../confirmation-views/GenericView'
+import { NormalizedSettingsChangeTransaction } from './types'
+interface ConfirmationViewProps {
+  txDetails: TransactionDetails
+}
+
+export function ConfirmationView({ txDetails }: ConfirmationViewProps) {
+  const confirmationViewType = getTransactionType({ txInfo: txDetails.txInfo })
+
+  switch (confirmationViewType) {
+    case ETxType.TOKEN_TRANSFER:
+      return (
+        <QueueSummary
```

Comment:
Why is this called QueueSummary? I would expect something like TokenTransfer.

Reply:
Just added this name to follow the same name we have on figma screen. Agree, TokenTransfer makes more sense